kubecost/values.yaml

```diff
@@ -1119,39 +1120,36 @@ cloudCost:
   # refreshRateHours:
   # queryWindowDays:
   # runWindowDays:
-  # serviceAccountName:
+
+  ## by default, the cloud cost pod will use the same service account as the aggregator pod.
+  ## this can be overridden to use a different service account.
+  serviceAccountName: ""
+
   readinessProbe:
     enabled: true
     initialDelaySeconds: 10
     periodSeconds: 10
     failureThreshold: 200
 
   ## Add a nodeSelector for aggregator cloud costs
-  # nodeSelector: {}
+  nodeSelector: {}
 
   ## Tolerations for the aggregator cloud costs
-  # tolerations: []
+  tolerations: []
 
   ## Affinity for the aggregator cloud costs
-  # affinity: {}
-
-  ## ServiceAccount for the aggregator cloud costs
-  # serviceAccountName: ""
+  affinity: {}
 
   ## Define environment variables for cloud cost
-  # env: {}
+  extraEnv: []
+  # - name: EXTRA_ENV_VAR
+  #   value: "extra_env_var_value"
 
   ## Define extra volumes for the cloud cost pod
-  # extraVolumes: []
+  extraVolumes: []
 
   ## Define extra volumemounts for the cloud cost pod
-  # extraVolumeMounts: []
-
-  ## Configure the Collections service for aggregator.
-  # collections:
-  #   cache:
-  #     enabled: false
```

Comment:
I think you deleted this config by accident?

https://github.com/kubecost/kubecost/blob/246679b5a5b07d92ecbd7dad1ffefac973c41179/kubecost/templates/aggregator/aggregator-statefulset.yaml#L552-L555

Reply:
my bad, and great catch!!!

Reply:
i see now. 
```
  # collections:
  #   cache:
  #     enabled: false
```